### PR TITLE
OCM-10883 | feat: include new manifest fields

### PR DESCRIPTION
**Pull request by gdbranco**
OCM-10883

### CODE REVIEW

File: model/clusters_mgmt/v1/manifest_type.model [hunk 1]
```diff
@@ -18,4 +18,16 @@ limitations under the License.
 class Manifest {
 	// List of k8s objects to deploy on a hosted cluster.
 	Workloads []Interface
+
+	// Spec of Manifest Work object from open cluster management.
```
Comment (andreadecorte) [inline model/clusters_mgmt/v1/manifest_type.model:22]: Useful also to add a link to https://open-cluster-management.io/concepts/manifestwork/

File: model/clusters_mgmt/v1/manifest_type.model [hunk 2]
```diff
@@ -18,4 +18,16 @@ limitations under the License.
...
+	Spec Interface
+
+	// Date and time when the manifest got created in clusters-service.
```
Comment (andreadecorte) [inline model/clusters_mgmt/v1/manifest_type.model:25]: curious, do we already expose this in the API? Or should we keep something generic like in OCM DB

File: model/clusters_mgmt/v1/manifest_type.model (hunk 1 above)
Comment (gdbranco) [reply model/clusters_mgmt/v1/manifest_type.model]: Done, thanks!

File: model/clusters_mgmt/v1/manifest_type.model (hunk 2 above)
Comment (gdbranco) [reply model/clusters_mgmt/v1/manifest_type.model:26]: What do you mean? We usually expose for other services, not sure why we wouldn't here

**Review by ciaranRoche — APPROVED** [review]
/lgtm

**Review by andreadecorte — APPROVED** [review]
lgtm

**Pull request merged**